generators/php/sdk/src/endpoint/request/WrappedEndpointRequest.ts

```diff
@@ -120,6 +134,48 @@ export class WrappedEndpointRequest extends EndpointRequest {
         writer.writeNodeStatement(this.stringify({ reference: header.valueType, name: header.name.name }));
     }
 
+    private writeBodyParameter(writer: php.Writer, valueAssignment: php.AstNode, propertyName: string): void {
```

Comment:
Given that this is specific to multipart, let's name this `writeMultipartBodyParameter`.

Reply:
done